setup.cfg

```diff
@@ -71,6 +71,7 @@ addopts = -W always --cov --cov-report=term-missing
 [check-manifest]
 ignore-bad-ideas =
     src/colander/locale/*
+    *.mo
```

Comment:
This shouldn't be necessary, all of those files should live in `src/colander/locale/*`, what made you add this to this PR?

Reply:
Ah, it's because this repo didn't chase the update to `check-manifest` that updates the glob matching, in this case, the line above it needs to run:

`src/colander/locale/**/*`

I've pushed a change for this directly to master, you may drop this change from your PR please.